.circleci/config.yml

```diff
@@ -317,10 +315,20 @@ jobs:
           command: |
             brew install cmake ninja
       - run:
-          name: Run CocoaPods installation and test framework
+          name: Install CocoaPods and build the framework
           command: |
             cd test/ApplePlatformsIntegrationTestApp
-            ./run.sh
+            pod install
+      - run:
+          name: Test MacOS application
+          command: |
+            . ./run.sh
+            test_apple_target "ApplePlatformsIntegrationTestApp"
+      - run:
+          name: Test iPhone application
+          command: |
+            . ./run.sh
+            test_apple_target "ApplePlatformsIntegrationMobileTestApp"
```

Comment:
While this test runs just fine, it will fail. I don't know how to assert the output yet. Please check the comment in the `run.sh` file.